web-app/src/pages/Login.vue

```diff
@@ -34,11 +34,30 @@
 
 <script lang="ts">
 import { Vue, Component } from 'vue-property-decorator';
+import axios, { AxiosResponse } from 'axios';
 
 @Component
 export default class Login extends Vue {
-  loginWithMS() {
-    console.log('login');
+  private async loginWithMS() {
+    const authUrlResponse: AxiosResponse = await axios({
+      method: 'POST',
+      url: 'http://localhost:3000/v1/users/login',
+    });
+    // eslint-disable-next-line @typescript-eslint/no-unsafe-member-access
+    window.location = authUrlResponse.data.data as Location;
```

Comment:
What do you do here? `window.location` is read-only property, anyway do you want to change url/page?

Reply:
`window.location` is redirecting page to microsoft login page. `authUrlResponse.data.data` contains url to this page.

Reply:
I think server should redirect the client to this URL. 
```javascript
res.redirect('https://app.example.io');
```

Reply:
Tried that. Didn't work.